app/src/components/health.js

```diff
@@ -2,41 +2,28 @@ const log = require('npmlog');
 
 const EmailConnection = require('../services/emailConn');
 
-const checks = {
+const healthCheck = {
   /** Checks the connectivity of the SMTP host
    *  @returns A result object
    */
-  getSmtpStatus: async () => {
-
-    const emailConnection = new EmailConnection();
-
-    const result = {
-      authenticated: false,
-      authorized: false,
-      endpoint: `https://${emailConnection.host}`,
-      healthCheck: false,
-      name: 'SMTP Endpoint'
-    };
-
+  getSmtpHealth: async () => {
+    let result = {name: 'smtp', healthy: false, info: null};
     try {
-      const emailConnectionOk = await emailConnection.checkConnection();
-
-      result.authenticated = true;
-      result.authorized = true;
-      result.healthCheck = emailConnectionOk;
+      const emailConnection = new EmailConnection();
```

Comment:
This should probably be moved outside the function to be consistent with the other components.